mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -314,109 +328,73 @@ export const Columns = (props: ColumnsProps) => {
-            if (i === index) {
-              return text;
+        const updateComment = async (comment: Comment, text: string) => {
+          await CommentDAL.updateComment(comment, text);
+          const updatedComments = row.original.comments.map((item) => {
+            if (item.uuid === comment.uuid) {
+              return new Comment({
+                ...comment,
+                description: text,
+              });
             }
 
             return item;
           });
 
-          const updatedDayReport = {...row.original, studentComments: updatedStudentComments};
-          updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
-        };
-
-        return (
-          <TableCell
-            buttonValue="add comment"
-            onButtonClick={() => createStudentComment()}
-          >
-            {row.original.studentComments
-              .map((studentComment, index) => (
-                <CellItem key={index}>
-                  <EditableText
-                    text={studentComment}
-                    onChangeFinish={(text) => updateStudentComment(text, index)}
-                  />
-                </CellItem>
-              ),
-              )}
-          </TableCell>
-        );
-      },
-    }),
-    columnHelper.accessor("mentorComments", {
-      header: "Mentor comments",
-
-      /**
-       * Cell with MentorComments items
-       */
-      cell: ({row}) => {
-
-        /**
-         * Create MentorComment
-         */
-        const createMentorComment = async () => {
-          const mentorComment = await MentorCommentDAL.createMentorComment(row.original);
-          const mentorComments = [...row.original.mentorComments, mentorComment];
-          const updatedDayReport = {...row.original, mentorComments};
+          const updatedDayReport = {...row.original, comments: updatedComments};
           updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
         };
 
         /**
-         * Update MentorComment
+         * Get user name
          */
-        const updateMentorComment = async (mentorComment: MentorComment, text: string) => {
-          await MentorCommentDAL.updateMentorComment(mentorComment, text);
-          const updatedMentorComments = row.original.mentorComments.map((item) => {
-            if (item.uuid === mentorComment.uuid) {
-              return new MentorComment({
-                ...mentorComment,
-                description: text,
-              });
-            }
-
-            return item;
-          });
-
-          const updatedDayReport = {...row.original, mentorComments: updatedMentorComments};
-          updateDayReportState(props.dayReports, props.setDayReports, updatedDayReport);
+        const getUserName = (users: Map<string, UserPreview>, uuid: string) => {
+          const user = users.get(uuid);
+          if (!user) {
+            return "unregistered user";
```

Comment:
get rid of strange magic values. let's forbid for unregistered users make comments